opentech/public/urls.py

```diff
@@ -1,10 +1,13 @@
 from django.urls import include, path
 
+from .news import views as news_views
 from .search import views as search_views
 from .mailchimp import urls as newsletter_urls
 
 
 urlpatterns = [
     path('search/', search_views.search, name='search'),
+    path('news/feed/', news_views.NewsFeed(), name='news_feed'),
+    path('news/<int:type>/feed/', news_views.NewsTypesFeed(), name='news_types_feed'),
```

Comment:
We can use `news_type` here and other related places as `type` to make it more explicit and also type is a python built-in method.